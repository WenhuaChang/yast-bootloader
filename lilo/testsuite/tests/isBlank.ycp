{
    include "testsuite.ycp";
    include "lilo/routines.ycp";

    TEST(``(isBlank("A")), [], nil);
    TEST(``(isBlank("AB")), [], nil);
    TEST(``(isBlank(" ")), [], nil);
    TEST(``(isBlank("\t")), [], nil);
    TEST(``(isBlank("\n")), [], nil);

}
