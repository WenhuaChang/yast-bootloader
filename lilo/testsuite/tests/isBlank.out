Return	false
Return	false
Return	true
Return	true
Return	true
